core/Cargo.toml

```diff
@@ -163,6 +163,7 @@ services-moka = ["dep:moka"]
 services-mongodb = ["dep:mongodb"]
 services-monoiofs = ["dep:monoio", "dep:flume"]
 services-mysql = ["dep:sqlx", "sqlx?/mysql"]
+services-nebulagraph = ["dep:rust-nebula", "dep:bb8", "dep:snowflaked"]
```

Comment:
OpenDAL has strict naming pattern for service names.

Please either use:

- `nebulagraph` / `Nebulagraph`

or

- `nebula_graph` / `NebulaGraph`

In every place this name showed up.